templates/nlu_evaluation_data/templates.yaml

```diff
@@ -0,0 +1,171 @@
+dataset: nlu_evaluation_data
+templates:
+  352e2261-149c-4363-a1e6-8fa4d5c93481: !Template
+    id: 352e2261-149c-4363-a1e6-8fa4d5c93481
+    jinja: "Given the following text, what is the scenario ? \n{{text}}\n|||\n{{\n\
```

Comment:
Use as little "linguistic terminology" as possible. You should replace the word "scenario" with something more explicit (layman's terms)! `nlu_eval_data_2` is a great example!